### PR TITLE
API guideline added

### PR DESCRIPTION
**Pull request by leila-tk**
Readme & Best Practice for API added

### CODE REVIEW

**Comment by jjercTK** [comment]
@leila-tk can we add a link in the main `README.md` that links to your changes?
Also, I think it will be good idea to add some thanks to the original author (I know we customized some things) and maybe link to the original source

**Comment by jjercTK** [comment]
I've just noticed there's a `Extracted from Thoughbot.` there. Im referring to the REST API part.

File: best-practices/README.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
+# Best Practices
+A guide for programming well.
+Extracted from Thoughbot.
```
Comment (jopereyral) [inline best-practices/README.md]:
```suggestion
Extracted from [Thoughbot](https://github.com/thoughtbot/guides).
```

Comment (jopereyral) [reply best-practices/README.md]: @jjercTK  do you think this is enough to reference the original author?

Comment (jjercTK) [reply best-practices/README.md]: i was referring to the source of the guidelines of an API: for GET, POST, PUT

Comment (jopereyral) [reply best-practices/README.md]: sure, it's no from thoughbot :(

**Comment by jjercx** [comment]
@leila-tk @jopereyral maybe we can add sources later, let's merge it for now and leave an issue

**Comment by leila-tk** [comment]
@jjercTK @jopereyral hey guys, just to clarify, I added **Extracted from thoughbot** only in the 'best practices' section because the entire API guideline is from other sources. That's why above the 'REST APIs' title says **Extracted from the work on some blogs**.

**Comment by jjercTK** [comment]
Yeah, i would prefer to be explicit with those "some blogs", but later maybe. Created #2 for this.

**Pull request merged**